core/src/main/java/org/apache/druid/common/config/JacksonConfigManager.java

```diff
@@ -50,6 +52,7 @@ public JacksonConfigManager(
     this.configManager = configManager;
     this.jsonMapper = jsonMapper;
     this.auditManager = auditManager;
+    this.jsonMapperSkipNull = jsonMapper.copy().setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
I think this object should be bound in the `JacksonModule` as a singleton.

Even though JacksonConfigManager is a singleton, if this binding changes in the future, we could inadvertently create an ObjectMapper per JacksonConfigManager instead of a singleton that can be shared across the process.
Otherwise, each time we create a new JacksonConfigMapper, we will create a new ObjectMapper, even though, jsonMapper is a singleton.

Reply:
Done